tests/tools/nnpackage_run/src/rawformatter.cc

```diff
@@ -72,12 +74,6 @@ void RawFormatter::dumpOutputs(const std::string &filename, std::vector<Allocati
 {
   uint32_t num_outputs;
   NNPR_ENSURE_STATUS(nnfw_output_size(session_, &num_outputs));
-  // TODO: Support multiple outputs
-  // Available options are same.
-  if (num_outputs != 1)
-  {
-    throw std::runtime_error("Only 1 output is supported for raw input");
-  }
```

Comment:
@hseok-oh You don't need to update existing output by this PR. It just removes throwing exception, which was not tested yet. It is tested in my local since the model is not a public model.